### DISCUSSION TITLE
Continuous Auto-Rejection Due to Profile - Homeschool

### DISCUSSION THREAD

**Discussion by cmontigel** [community/community#51343] [GitHub Education] [answered]
Hello! I am trying to apply for the student developer pack and keep getting rejected within minutes. I am attempting to submit my transcript and upon submission, I receive the note below.

![image](https://user-images.githubusercontent.com/67985645/228585499-c6e4ad9f-5bed-4050-894a-cbf3c302d48f.png)

I have filled out my profile (and it is marked as public), however, I continued to get the same response. I tried reaching out directly to GitHub Support but my ticket got closed by support without a resolution after I reopened the automatic closing.

Does anyone know how to get around this? It has been quite a painful process to apply for the pack as a homeschool student.

**Comment by samus-aran** [comment, marked as answer]
Hi @Astr0clad 👋 

I can see you have updated their Profile details so I would suggest that you could try logging out of GitHub and logging back in before trying to resubmit. Let us know how you get on and share your rejection reasons 👍

**Reply by cmontigel** [reply]
Switching to a mobile device and using the higher quality camera (including signing in and out) seems to have fixed it. Thank you so much!

**Reply by samus-aran** [reply]
Awesome to hear that @Astr0clad 🎉 Plus getting back to us helps others with similar applications 👍 

Enjoy exploring GitHub Global Campus:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋
